hooks/scan_scene_tk-mari.py

```diff
@@ -71,13 +71,16 @@ def execute(self, **kwargs):
 
                 params = {"geo":geo.name(), "channel":channel.name()}
 
+                # find all publishable layers:
+                publishable_layers = self.find_publishable_layers_r(channel.layerList())
+                if not publishable_layers:
+                    # no layers to publish!
```

Comment:
This is actually allowed but the publisher was previously showing that it could do something it couldn't - this just fixes that so I think should be ok